source/integrate_with_govuk_pay/index.html.md.erb

```diff
@@ -69,6 +69,8 @@ using the API.
 
 For example, you could automatically fetch data about the outcome of payment journeys. You could import that into your finance system so payments can be reconciled against bank transaction information.
 
+To help you reconcile, you can [add custom metadata to payments](/optional_features/custom_metadata).
```

Comment:
Can I suggest: 
```You also have the option of adding custom metadata to payments if you wish to reconcile payments using your own defined codes.```

Reply:
Thank you - I've tweaked for style. Does this still capture it?
`You can also add custom metadata to payments if you want to reconcile payments using your own codes.`

Reply:
👍 sounds good!